tests/test_views_admin.py

```diff
@@ -0,0 +1,242 @@
+from pyramid.config import Configurator
+import unittest
+
+from kinto.core import events
+from kinto.core.storage import exceptions
+from kinto.core.testing import get_user_headers
+from kinto.core import utils
+from kinto.views.admin import get_parent_uri
+
+from .support import BaseWebTest, MINIMALIST_BUCKET, MINIMALIST_COLLECTION, MINIMALIST_RECORD
+
+
+class DeleteUserDataTest(BaseWebTest, unittest.TestCase):
+    doomed_bucket_url = "/buckets/user_to_delete"
+    doomed_collection_url = doomed_bucket_url + "/collections/collection_to_delete"
+    doomed_bucket_collection_url = doomed_bucket_url + "/collections/collection"
+    safe_bucket_url = "/buckets/user_safe"
+    safe_bucket_doomed_collection_url = safe_bucket_url + "/collections/collection_to_delete"
+    safe_collection_url = safe_bucket_url + "/collections/collection_safe"
+    safe_collection_doomed_record_url = safe_collection_url + "/records/record_to_delete"
+    safe_record_url = safe_collection_url + "/records/record_safe"
+    safe_group_containing_doomed_user = safe_bucket_url + "/groups/silly-group"
+    shared_bucket_url = "/buckets/both"
+    doomed_user = get_user_headers("doomed", "trustno1")
+    hmac_secret = "secret_hmac_for_userids"
+    user_hash = utils.hmac_digest(hmac_secret, "mat:secret")
+    user_principal = f"basicauth:{user_hash}"
+    doomed_hash = utils.hmac_digest(hmac_secret, "doomed:trustno1")
+    doomed_user_principal = f"basicauth:{doomed_hash}"
+    delete_user_url = f"/__user_data__/{doomed_user_principal}"
+    subscribed = (events.ResourceChanged,)
```

Comment:
Maybe this doesn't have to be a class attribute IDK

Reply:
I was basing this on the code in `test_events`, although I admit it's a little excessive for this case. There might be a larger solution to cleaning up how we initialize apps in the test suites generally..